microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
+
+Employ User-Agent Client Hints when you want to:
+- Determine if new browser activity is from the expected user.
+- Customize hints or instructions if the user is new to this site.
+
+Do not use User-Agent Client Hints to:
+- Block *unsupported* browsers.
+- Restrict access to features on your site.
+
+### User Agent Client Hints HTTPS header
+
+When a browser requests a web page from a server, it sends an HTTPS client header request. The server replies with a specific set of HTTPS response header data. User-Agent Client Hints automatically return the brand and mobile attributes. It can easily include additional data.
 
 By default, Chromium browsers including Microsoft Edge send the `Accept-CH-UA` response header in the following format.  
 
 ```https
-Sec-CH-UA: "Chromium";v="91", "Microsoft Edge";v="91","Dummy;Browser Brand";v="99"
+Sec-CH-UA: "Chromium";v="92", "Microsoft Edge";v="92","Placeholder;Browser Brand";v="99"
 Sec-CH-UA-Mobile: ?0
 ```  
 
-> [!NOTE]
-> Client Hints are only sent over secure connections using `HTTPS`.  
-
-The following low entropy hints are sent by default.  If you need to access more information, send one of the following request headers.  
+The following table shows the available header request hints with sample responses. For example, to discover information such as operating system, include `Sec-CH-UA-Platform` in the header request.
+  
 
-#### User-Agent request and response headers  
-
-| Request header | Example response value |  
+| User-Agent Request header | Example User-Agent response value |  
 |:--- |:--- |  
 | `Sec-CH-UA` | `"Chromium";v="91", "Microsoft Edge";v="91","GREASE";v="99"` |  
 | `Sec-CH-UA-Mobile` | `false` |  
 | `Sec-CH-UA-Full-Version` | `91.0.866.0` |  
 | `Sec-CH-UA-Platform` | `Windows` |  
 | `Sec-CH-UA Platform-Version` | `10.0` |  
 | `Sec-CH-UA-Arch` | `x86` |  
-| `Sec-CH-UA-Model` |  |  
+| `Sec-CH-UA-Model` | `Surface Pro` |  
+
+> [!NOTE]
+> User-Agent Client Hints are only sent over secure connections using `HTTPS`.
 
-### User-Agent Client Hints JavaScript API  
+### User Agent Client Hints JavaScript API  
 
-The default response value from `navigator.userAgentData` uses the following format.  
+You can access User-Agent Client Hints using Javascript on the client side. When you call the default `navigator.userAgentData` it returns the following response.  
 
 ```javascript
 { brands: [ {brand: "Chromium","version":"91"}, {brand: "Microsoft Edge","version":"91"}, {brand: "GREASE","version":"99"}, ]
 mobile: false }
 ```  
 
-If you need to access more detailed information, use the [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues] method.  
+To help you discover issues such as hardcoded brand names, Microsoft Edge includes a `GREASE` brand value that changes over time and is formatted to break if it detects common string parsing issues.

```

Comment:
> Microsoft Edge includes a `GREASE` brand value that changes over time and is formatted to break if it detects common string parsing issues.

We're not able to detect string parsing issues. The purpose of a changing GREASE value is to break other people's potentially "incorrect" usage of Client Hints. For example, we don't want people to detect Edge version 91 by checking if the brands list is exactly:
`{brand: "Chromium","version":"91"}, {brand: "Microsoft Edge","version":"91"}, {brand: "GREASE","version":"99"}`
Rather, we want them to detect Edge version 91 if and only if there is a brand in that list that is "Microsoft Edge" and the corresponding version is "91", meaning something like this would pass an Edge 91 check too:
`{brand: "Microsoft Edge","version":"91"}, {brand: "Chromium","version":"91"}, {brand: "BROWSER","version":"100"}`

So, I think this should read more like:

> Microsoft Edge includes a `GREASE` brand value that changes over time. Its purpose is to prevent sites from attempting to match the entire brand list when attempting to detect a version of Edge.

Reply:
fixed